types/mapbox__mapbox-gl-draw/index.d.ts

```diff
@@ -0,0 +1,123 @@
+// Type definitions for @mapbox/mapbox-gl-draw 1.2
+// Project: https://github.com/mapbox/mapbox-gl-draw
+// Definitions by: Tudor Gergely <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/mapbox__mapbox-gl-draw/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me